### PR TITLE
[docs] update staticClusterConfiguration docs, add DVP specifics

### PR DESCRIPTION
**Pull request by Suselz**
## Description
This update adds to the documentation of the `StaticClusterConfiguration` object. The features of cluster deployment within Deckhouse Virtualization Platform (`DVP`) are specified.

## Why do we need it, and what problem does it solve?

The change is needed to clarify a particular condition where the 'internalNetworkCIDRs' parameter must be filled. This is necessary if the cluster nodes have more than one network interface, or the cluster is being deployed inside DVP. This clarity will help users to avoid potential issues during deployment.

## What is the expected result?
After applying these changes, the documentation of `StaticClusterConfiguration` will be clearer to users deploying inside DVP. 

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: fix 
summary:  Update `staticClusterConfiguration` docs, add Deckhouse Virtualization Platform deployment specifics.
impact_level: low
```

### CODE REVIEW

File: candi/openapi/doc-ru-static_cluster_configuration.yaml [hunk 1]
```diff
@@ -19,4 +19,8 @@ apiVersions:
 
           Внутренние сети используются для связи компонентов Kubernetes (`kube-apiserver`, `kubelet` и т. д.) между собой.
 
-          Если каждый узел в кластере имеет только один сетевой интерфейс, параметр можно не указывать и ресурс `StaticClusterConfiguration` можно не создавать.
+          Параметр обязателен к заполнению, если:
+
+            -  Узлы кластера имеют более одного сетевого интерфейса
+
+            -  Развертывание кластера происходит внутри Deckhouse Virtualization Platform
```
Comment (z9r5) [inline candi/openapi/doc-ru-static_cluster_configuration.yaml:26]:
```suggestion
          Параметр обязателен к заполнению, если:
            -  Узлы кластера имеют более одного сетевого интерфейса
            -  Развертывание кластера происходит внутри Deckhouse Virtualization Platform
```

**Comment by deckhouse-BOaTswain** [comment]
Failure: Cherry pick [PR](https://github.com/deckhouse/deckhouse/pull/10088) was created but cannot be merged

**Comment by deckhouse-BOaTswain** [comment]
Backport failed. See [Job](https://github.com/deckhouse/deckhouse/actions/runs/11274325710) for details.

**Comment by z9r5** [comment]
Cherry-picked into release-1.64

**Pull request merged**